pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -553,14 +558,15 @@ func ipSecReplaceStateOut(localIP, remoteIP net.IP, nodeID uint16, localBootID,
 	return key.Spi, xfrmStateReplace(state, remoteRebooted)
 }
 
-func _ipSecReplacePolicyInFwd(src, dst *net.IPNet, tmplSrc, tmplDst net.IP, proxyMark bool, dir netlink.Dir) error {
+func _ipSecReplacePolicyInFwd(src, dst *net.IPNet, tmplSrc, tmplDst net.IP, proxyMark bool, dir netlink.Dir, reqID int) error {
```

Comment:
nit/nonblocking/suggestion: while we're changing this, we might as well get rid of the C++ style underscore naming here, doesn't mean much in Go.

Reply:
There are a couple of styling improvements required - removing underscore and passing a struct as a parameter since param list is getting long. I was thinking of doing this as an independent PR, so code style can be the focus of that review.